test/Microsoft.DocAsCode.MarkdigEngine.Tests/TripleColonTest.cs

```diff
@@ -25,5 +27,57 @@ public void TripleColonTestGeneral()
 
             TestUtility.AssertEqual(expected, source, TestUtility.MarkupWithoutSourceInfo);
         }
+
+        [Fact]
+        public void TripleColonTestNested()
```

Comment:
This test have no `Assert`. Do we need this one for now? Or add it when the feature is ready?

Reply:
@jeahyoun Please add some valid test cases for these two changes.

Reply:
I'll check into this - this code was commented out when I found it. I'll ask and see if it's necessary.

Reply:
I've removed the unused test; it appears that even if it did have a valid assertion, since the moniker range block does not use the triplecolon extension, the test would not prove that nesting triple colon blocks works correctly. We only have the zone extension (which does not allow nesting) and the Chromeless Forms extension (which is a self-closing block) so we currently do not have this use case.

I looked into a way to test the "Closed" properties of the blocks, but I don't think it's possible to access the block properties in the tests - it seem that we can only check input and output from Markdig, not the internal state of the processor. If there is a way to do this, I would be interested.

Reply:
Would you try to get the  MarkdownDocument use this method? https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.MarkdigEngine/MarkdigMarkdownService.cs#L70

I think you can get the block and test "Closed" properties with MarkdownDocument .

Reply:
Added a test to check the block conditions for TripleColon.